src/config.ts

```diff
@@ -23,8 +23,11 @@ export const VideoProgressMinMax = {
 
 export const PLAYLIST_LIMIT = 25;
 
-// The externalData attribute of Cleeng can contain max 5000 characters
-export const MAX_WATCHLIST_ITEMS_COUNT = 48;
+export const MAX_WATCHLIST_ITEMS_COUNT = {
+  CLEENG: 38, // The 'externalData' attribute of Cleeng can contain max 4000 characters
+  JWP: 50, // Limit of media_ids length passed to the /apps/watchlists endpoint
+  DEFAULT: 38, // Local storage
+};
```

Comment:
Shouldn't this be defined in the integration? I'm hoping we can remove all integration references from the non-service code.

Reply:
Will add it to integrations.